src/disclosures/js/index.js

```diff
@@ -49,9 +49,7 @@ var app = {
 
               // Update expenses model bases on region and salary
               region = schoolValues.BLSAverage.substr( 0, 2 );
-              salary = schoolValues.medianSalary;
-              expensesModel.resetCurrentValues( region, salary );
-              expensesView.updateView( getExpenses.values() );
+              $( '#bls-region-select' ).val( region ).change();
```

Comment:
Please fix/remove the following variables that are throwing new `lint` warnings for this file:

> 9:5   warning  'getExpenses' is defined but never used  no-unused-vars
  38:19  warning  'salary' is defined but never used       no-unused-vars